helm/prometheus/values.yaml

```diff
@@ -15,6 +15,10 @@ config:
   specifiedInValues: true
   value: {}
 
+## Enable access to prometheus web admin API
+##
+enableAdminApi: true
```

Comment:
the helm charts are no longer maintained here, we are about to remove them from this repository

Reply:
Should I take out the changes this PR makes to the helm chart as it is no longer maintained?

Reply:
Yeah, let's remove these here